section2/reflection.md

```diff
@@ -2,14 +2,26 @@
 
 1. What are two points from the Growth Mindset article and/or video that either resonated with you, or were brand new to you?
 
+I like the SMART acronym. It will help me be able to break projects down into realistic chunks to work on.
+The reading also helped to reinforce that it is ok to ask for help.
+
 2. In which ways do you currently demonstrate a Growth Mindset? In which ways do you _not_?
 
+I've always had an issue asking for help, because I've always been expected to just know the answer, and been treated poorly for asking.
+I also need to realize that mistakes are part of the learning, and stop trying to be a perfectionist.
+
 3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
 
+I think that with the work that we've been doing with each session and each section of Module 0, it is reinforcing the habits we are developing, so it did not throw me off.
+
 4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
+A conditional statement is a statement that if a particular trigger happens, then something else can happen. An example of this could be online status in Slack. If you're logged in, it will show you online, if not, it will show you offline.
 
 5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
+You can add multiple conditions with an `else` trigger. This would be like stating if I have $3.50, I can pay the Loch Ness Monster, else she eats me.
 
 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
+You can use logical operator && to make sure that TWO conditions are met, and || to make sure that one of the two conditions is met.
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+7. What questions do you still have about `if` statements and/or functions?
+I'd like to see how to code a statement that requires a combination of "These two conditions must be met, or the first condition and one of conditions 3&4 must be met."
```

Comment:
I'm not 100% sure what you're asking, but possibly something like this...
`if ((cond1 && cond2) || (cond1 && cond3) || (cond1 && cond4))`